index.html

```diff
@@ -56,6 +56,127 @@ <h1>
         </li>
     </ul>
 </div>
+<div class="readmore-button">
+    <a>
+        <img src="./images/icons/readmore-icon.svg" alt="read more ">
+    </a>
+</div>
+</section>
+<section id="works" >
+    <div class="recent-works">
+        <h2>My Recent </br> Works</h2>
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div>
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div> 
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div> 
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
```

Comment:
Great job on your works section! I have some suggestions to make it even better and closer to the Figma template:
- [ ] Could you please check the `padding` and `border-radius` of your buttons? They're not quite the same as the template, and making these small changes will help your project look more polished. 🙌
### Your design:
![image](https://user-images.githubusercontent.com/107952724/235262207-215f6d34-7edb-47a3-bead-d3ff468aeadf.png)

### Figma design:
![image](https://user-images.githubusercontent.com/107952724/235262139-44f07091-2bfb-4100-bec5-a2b1feae5f88.png)

----
- [ ] Additionally, I noticed that your buttons are missing the `hover, enabled, and pressed` effects that the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=yCafCmU4xSO1HuYd-0) includes. If you could add these effects using the table provided in the template, it will make your project look even more professional. 👌
### Button Effects
![image](https://user-images.githubusercontent.com/107952724/235262038-495d2928-63f1-40d1-b564-29a59c05aa62.png)

  **_Please apply these suggestions to all of the buttons inside your project._** 😉